drivers/bridge/setup_device.go

```diff
@@ -31,7 +32,29 @@ func setupDevice(config *NetworkConfiguration, i *bridgeInterface) error {
 	}
 
 	// Call out to netlink to create the device.
-	return netlink.LinkAdd(i.Link)
+	if err = netlink.LinkAdd(i.Link); err != nil {
+		return types.InternalErrorf("Failed to program bridge link: %s", err.Error())
+	}
+
+	if config.FixedCIDRv6 != nil {
```

Comment:
This `if` clause should really belong in `setupFixedCIDRv6` of course without the if check because that function will only be called when `config.FixedCIDRv6 != nil` anyways